app/src/main/java/com/smartpack/kernelmanager/services/profile/Widget.java

```diff
@@ -74,7 +74,7 @@ private PendingIntent getPendingIntent(Context context, String action) {
         Intent intent = new Intent(context, getClass());
         intent.setAction(action);
         return PendingIntent.getBroadcast(context, 0, intent, Build.VERSION.SDK_INT >=
-                android.os.Build.VERSION_CODES.M ? PendingIntent.FLAG_IMMUTABLE : 0);
+                android.os.Build.VERSION_CODES.M ? PendingIntent.FLAG_MUTABLE : 0);
     }
```

Comment:
Since we have an exact action: `LIST_ITEM_CLICK` and the updated pending intent doesn't seem to be much different form the older one, maybe a combination of `PendingIntent.FLAG_IMMUTABLE | PendingIntent.FLAG_UPDATE_CURRENT`  should also do the job.

Could you test it?

Reply:
No, obviously it doesn't work. The FLAG_UPDATE_CURRENT flag only affects how PendingIntent.getBroadcast works, but the extras are filled in the widget when the item is clicked, and setPendingIntentTemplate + setOnClickFillInIntent needs PendingIntent.FLAG_MUTABLE to work, no? I'm not an Android developer so I could be wrong. But the tests confirmed that PendingIntent.FLAG_IMMUTABLE | PendingIntent.FLAG_UPDATE_CURRENT doesn't works here.

Reply:
@nikoyak @Lennoard 
I don't have much ways to test it except on an emulator. I'm, anyway, merging this PR. Thanks